tools/build/node-alpine.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+set -e
+
+# Node installation for Alpine systems
+apk update
+
+# Get exact package version where rN isn't known
+NODE_APK_REV=$(apk search nodejs | grep nodejs-$NODE_VERSION | rev | cut -d'-' -f1 | rev )
+NODE_APK=nodejs@edge=$NODE_VERSION-$NODE_APK_REV
+
+# Intentionally using NODE version for NPM
+NPM_APK_REV=$(apk search nodejs-npm | grep nodejs-npm-$NODE_VERSION | rev | cut -d'-' -f1 | rev )
+NPM_APK=nodejs-npm@edge=$NODE_VERSION-$NPM_APK_REV
+
+echo $NODE_APK
+echo $NPM_APK
+
+apk add python2 libuv@edge $NODE_APK $NPM_APK
+
+npm install npm@"$NPM_VERSION"
+npm install gulp bower
+
+apk del python2
```

Comment:
also - does this delete python2? wouldn't it be needed later for installing all other packages like node-sass and what not? or is it just clearing the index or smth

i'm not very good at linux

Reply:
You make a good point. I should double check this